class/class-form.php

```diff
@@ -0,0 +1,340 @@
+<?php
+/**
+ * @package    Feature
+ */
+
+/**
+ * Form section
+ */
+class Form {
+
+	/**
+	 * Add new idea form
+	 */
+	public static function add_new_idea_form() {
+		if ( ! isset( $_POST['new-idea-form-nonce'] ) || ! wp_verify_nonce( $_POST['new-idea-form-nonce'], 'new-idea-form' ) ) {
+			return;
+		}
+		if ( isset( $_POST['add_new_feature'] ) ) {
+			$project_name     = $_POST['project_name'];
```

Comment:
Every dynamic inputs from $_GET or $_POST should be sanitized. Also it is better to have isset() checking on GET and POST request data.